### PR TITLE
Fix Monstrous War-Leech

### PR DESCRIPTION
**Pull request by Northmoc**
image

### CODE REVIEW

File: forge-game/src/main/java/forge/game/ability/AbilityUtils.java [hunk 1]
```diff
@@ -3566,6 +3566,7 @@ public static int objectXCount(final List<?> objects, final String s, final Card
      */
     public static int handlePaid(final Iterable<Card> paidList, final String string, final Card source, CardTraitBase ctb) {
         if (paidList == null) {
+            //issue is here for Monstrous War-Leech
```
Comment (Northmoc) [inline forge-game/src/main/java/forge/game/ability/AbilityUtils.java:3569]: The problem is below I think because of `.`

**Comment by Hanmac** [comment]
i need to debug itself what is wrong with this

**Comment by tool4ever** [comment]
The problem is those Aggregates.min/max have the Integer limits as default

I think also entering there for `paidList.isEmpty()` should be safe

**Comment by Hanmac** [comment]
> The problem is those Aggregates.min/max have the Integer limits as default
> 
> I think also entering there for `paidList.isEmpty()` should be safe

iterable doesn't has `isEmpty`, we need `Iterables.isEmpty(paidList)`

**Review by Hanmac — APPROVED** [review]
👍

**Pull request merged**